ArduPlane/is_flying.cpp

```diff
@@ -35,7 +35,7 @@ void Plane::update_is_flying_5Hz(void)
     }
 
 #if HAL_QUADPLANE_ENABLED
-    is_flying_bool = quadplane.is_flying();
+    is_flying_bool = quadplane.is_flying_vtol();
```

Comment:
This change matches this:

https://github.com/ArduPilot/ardupilot/blob/5db5bbfbb369c99896b6cf9decce942ab190bac2/ArduPlane/is_flying.cpp#L191-L193

Currently you can have a 0 `isFlyingProbability` but `plane.is_flying()` returns `true` due to the difference between ` is_flying_vtol()` and `is_flying()`. That can still happen of course, but now that will drag `isFlyingProbability` back up to 1.